openpype/hosts/maya/plugins/publish/collect_look.py

```diff
@@ -601,6 +601,9 @@ def collect_resources(self, node):
                                                      source,
                                                      computed_source))
 
+            if not source:
+                self.log.info("source is empty, skipping...")
+                continue
```

Comment:
Would it be worth adding a comment in code to state why we'd sometimes want to skip collecting on a file node.

Since technically with this change it also now allows a user to publish a look using a `file` node that doesn't have a texture set without that being validated against, yes?

Reply:
Doesnt Arnold throw an error if this happens? If so we should defo validate.

Reply:
I seems this is allowed only for Renderman as it seems. I can add logic just for that but I hate to have so much renderer specific codes....

Reply:
I'm not quite sure if it should be our responsibility to check if a file node is empty or not. I would just allow it across the board and leave it to the artist. It's way too wide of a problem to potentially deal with it for all renderers and their usecases.

Reply:
I've limited it to renderman nodes, I believe this is the safest and easiest approch to it